util/templates/defaults.yaml

```diff
@@ -107,6 +107,17 @@ params:
     type: float
     usages: ["pv"]
     advanced: true
+  - name: workmodesolax
```

Comment:
Definition belongs in the solax template as it's specific.

Reply:
move any to solax template